README.md

```diff
@@ -47,3 +47,7 @@ We do not want your answer to be publicly available on the Internet for anyone t
   * While GitHub charges for private repos, you can also use BitBucket or GitLab, both of which allow private repos on their free plans.
 * Send us an email with a zip or tarball attachment. You can send this to your HR contact at MaxMind.
 
+## Copyright
+
+This repository is copyright MaxMind 2014-2017.  You may distribute it under
```

Comment:
There are two spaces after the period here while the rest of the document has one space.